### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by vlados31999** [jertel/elastalert2#1446] [Q&A] [answered]
Tell me how you can run elast-alert with multiple configurations at the same time?

**Comment by jertel** [comment]
Please provide more information, or an example.

**Comment by jmacdone** [comment, marked as answer]
Are you talking about running with multiple rule configurations?
https://elastalert2.readthedocs.io/en/latest/ruletypes.html#common-configuration-options
> Every file that ends in .yaml in the rules_folder will be run by default
